### PR TITLE
Add a deprecation notice for this feature

### PR DESCRIPTION
**Pull request by mikeSimonson**
It can be replaced by comment feature of the doctrine
Type object.

### CODE REVIEW

**Comment by mikeSimonson** [comment]
@stof Should I put the env var SYMFONY_DEPRECATIONS_HELPER=weak on every build ?

File: ConnectionFactory.php [hunk 1]
```diff
@@ -84,6 +84,9 @@ private function initializeTypes()
                 Type::addType($type, $typeConfig['class']);
             }
             if ($typeConfig['commented']) {
+                @trigger_error(
+                    'The commented option of the type configuration is deprecated since DoctrineBundle 1.7.
+                    Use the feature of the Type class Instead', E_USER_DEPRECATED);
```
Comment (stof) [inline ConnectionFactory.php]: `instead` should not have a capital letter.

And you should not put a newline inside the string itself. If you want to use 2 lines in the source code, use concatenation.

Btw, I would add the `$type` in the message to make it easier to know which type should be updated.

Comment (mikeSimonson) [inline ConnectionFactory.php]: @stof Done. What about the tests that fail because of the deprecations errors ?

**Comment by stof** [comment]
@mikeSimonson No. you should add `@group legacy` on the test covering the BC layer (and ensuring other features covered by the same test are also covered by a non-legacy test)

File: ConnectionFactory.php [hunk 2]
```diff
@@ -84,6 +84,9 @@ private function initializeTypes()
                 Type::addType($type, $typeConfig['class']);
             }
             if ($typeConfig['commented']) {
+                @trigger_error(
+                    sprintf('The commented option of the type configuration is deprecated since DoctrineBundle 1.7.' .
+                    'Use the feature of the Type class instead. (in %s)', $type), E_USER_DEPRECATED);
```
Comment (stof) [inline ConnectionFactory.php]: I would say `Use the "requiresSQLCommentHint" method of the Type class instead`

**Comment by alcaeus** [comment]
Superseded by #947.

**Pull request closed without merging**